src/Vm.sol

```diff
@@ -259,6 +259,9 @@ interface VmSafe {
     function parseJsonBytes32(string calldata, string calldata) external returns (bytes32);
     function parseJsonBytes32Array(string calldata, string calldata) external returns (bytes32[] memory);
 
+    // Checks if a key exists in a JSON or TOML object.
+    function keyExists(string calldata, string calldata) external view returns (bool);
```

Comment:
We try to always used named inputs/outputs here for clarity, do you mind updating? (looks like the above `parseJson*` methods are also missing named params if you wanted to fix those also 😅

Reply:
oh yeah just followed what was above 😅 will update them all!

Reply:
hey @Evalir just bumping this